homeassistant/components/tuya/strings.json

```diff
@@ -517,6 +517,9 @@
       "total_energy": {
         "name": "Total energy"
       },
+      "total_reverse_energy": {
+        "name": "Total reverse energy"
```

Comment:
I don't think think this is a proper name that is generally reflected in Home Assistant. This is about importing energy, right?